mrc/configuration/configurator.py

```diff
@@ -0,0 +1,6 @@
+from mrc.shared.relativeposition import RelativePosition
+
+
+class Configurator:
+    def __init__(self, target_position: RelativePosition):
+        self.target_position = target_position
```

Comment:
We are asumming that position is not time-dependent, right? I'm not sure if we can do it that way but lets keep it for now.

Reply:
Adding a new functionality responsible for changing the position would be no problem, but in my opinion unit need its initial target from the beginning